nodejs/awsx/elasticloadbalancingv2/network.ts

```diff
@@ -144,7 +144,7 @@ export class NetworkListener
             protocol,
             loadBalancer,
             defaultActions,
-        }, { parent: loadBalancer, ...opts });
+        }, opts);
```

Comment:
Is there a reason we are changing this?

Reply:
In particular, it looks like we're making the opposite change to what we are doing elsewhere - like in `targetGroup`.

Reply:
Sorry, i can def see how that's unclear.  In both cases what i'm doing is trying to move this logic to a centralized location in the base class.  We were inconsistent with how we did this, and now i'm trying to do it in the same way across AppListener/NetworkListener/AppTargetGroup/NetworkTargetGroup.